docs/annexes/annex-06-pid-rulebook.md

```diff
@@ -70,19 +70,22 @@ attestation within the EUDI Wallet, as well as the PID metadata:
     -   Section 2.2.2 describes how Member States can specify national
         attributes by defining a domestic PID namespace.
 
--   Section 2.3 specifies the set of data elements covering the PID
+-   Section 2.3 specifies the SD-JWT VC Type to be used for PID
+    attestations in SD-JWT VC format.
```

Comment:
```suggestion
-   Section 2.3 specifies the SD-JWT VC Type to be used for PID
    in SD-JWT VC format.
```

Reply:
PID attestations is the term used in the rest of the document already